cypress/support/commands.ts

```diff
@@ -28,38 +28,44 @@ import '@testing-library/cypress/add-commands'
 declare global {
 	namespace Cypress {
 		interface Chainable {
-			/**
-			 * @description Custom command to select DOM element by data-testid attribute.
-			 * @example cy.getByTestID('settings')
-			 */
-			getByTestID (value: string): Chainable<Element>
-	
-			/**
-			 * @description Redirects to main page and creates an empty wallet for testing. Useful on starts of tests.
-			 * @param {boolean} [isRandom=false] default = false, creates randomly generated mnemonic seed or abandon x23
-			 * @example cy.createEmptyWallet(isRandom?: boolean)
-			 */
-			createEmptyWallet (isRandom?: boolean): Chainable<Element>
-	
-			/**
-			 * @description Sends UTXO DFI to wallet.
-			 * @example cy.sendDFItoWallet().wait(4000)
-			 */
-			sendDFItoWallet (): Chainable<Element>
-	
-			/**
-			 * @description Sends DFI Token to wallet.
-			 * @example cy.sendDFITokentoWallet().wait(4000)
-			 */
-			sendDFITokentoWallet (): Chainable<Element>
-	
-			/**
-			 * @description Sends token to wallet. Accepts a list of token symbols to be sent.
-			 * @param {string[]} tokens to be sent
-			 * @example cy.sendTokenToWallet(['BTC', 'ETH']).wait(4000)
-			 */
-			sendTokenToWallet (tokens: string[]): Chainable<Element>
-		}
+      /**
```

Comment:
Seems like the changes here is from `tab` to 2 `space` indentation, can you revert it?

Reply:
oh yeah good point, let me check